packages/create-amplify/src/execute_with_logger.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, it, mock } from 'node:test';
+import { executeWithLogger } from './execute_with_logger.js';
+import assert from 'assert';
+
+void describe(() => {
+  void it('executes a command with no args', async () => {
+    const execaMock = mock.fn();
+
+    // const executeWithLogger = new ExecuteWithLogger(
+    //     '/testProjectRoot',
+    //     execaMock as never
+    // );
```

Comment:
Can you remove all commented out code?
Commented out code is ok if there's TODO item with gh issue attached.

Reply:
Whoops, forgot to delete this after trying stuff out